spec/mailers/user_mailer_spec.rb

```diff
@@ -123,12 +123,6 @@
       expect(mail.html_part.body).to have_content(
         t('user_mailer.phone_changed.intro', app: APP_NAME)
       )
-
-      expect(mail.html_part.body).to have_content(
-        t('user_mailer.phone_changed.help',
-          app: APP_NAME,
-          contact_phone_number: CONTACT_PHONE_NUMBER)
-      )
```

Comment:
I think it's worth it to have a test that makes sure that the email contains links that point to the right place. We want to be sure the users will be able to get help.